tests/plotting/conftest.py

```diff
@@ -36,22 +36,31 @@ def _is_vtk(obj):
 
 
 @pytest.fixture()
+def skip_check_gc(check_gc):
+    """Skip check_gc fixture."""
+    check_gc.skip = True
+
+
+@pytest.fixture(autouse=True)
 def check_gc(request):
     """Ensure that all VTK objects are garbage-collected by Python."""
     gc.collect()
     before = {id(o) for o in gc.get_objects() if _is_vtk(o)}
-    yield
-
-    # Do not check for collection if the test session failed. Tests that fail
-    # also fail to cleanup their resources and this makes reading the unit test
-    # output more difficult.
-    #
-    # This applies to the entire session, so it's going to be the most useful
-    # when debugging tests with `pytest -x`
-    pyvista.close_all()
-    if request.session.testsfailed:
```

Comment:
This removed block to skip gc test using when any previous test failed (using `request.session.testsfailed` was added in #2941.  I needed to turn this off to easily find all the failing tests here, and did not notice any confusing output in doing so.  @akaszynski implemented this and maybe can shed some light on whether it should still be included.

Reply:
It seems that it my local pytest invocation is somehow different than the CI invocation and it does add confusing output for gc failures on the CI.  We might need to turn it back on.  Will do more digging.

Reply:
I was misinterpreting my local vs. CI pytest results.  When looking closer, I did not see anything more complicated after removing this if block for the failures.  Note that in #4500 only a small number of tests were identified as failing (see https://github.com/pyvista/pyvista/pull/4500#issuecomment-1581261637), but in reality a large number fail (mostly from within the tests/plotting/jupyter folder, tests/plotting/test_axes, and tests/plotting/test_charts).  The discrepancy is due to this if block.